### PR TITLE
ItemFrame.php

### PR DESCRIPTION
**Pull request by ghost**
Add ItemFrame.php

### CODE REVIEW

**Comment by robske110** [comment]
Hopefully not copied from IM or Genisys?

**Comment by MarcoDiGio** [comment]
It's from Genisys

**Comment by aodzip** [comment]
That means there should a tile

**Comment by DaPigGuy** [comment]
It's all in PHP7 too.

**Comment by PocketKiller** [comment]
And you bindly copy paste...

File: src/pocketmine/block/ItemFrame.php [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+			]);
+			Tile::createTile(Tile::ITEM_FRAME, $this->getLevel()->getChunk($this->x >> 4, $this->z >> 4), $nbt);
+		}
+
+		if($tile->getItem()->getId() === 0){
+			$tile->setItem(Item::get($item->getId(), $item->getDamage(), 1));
+			if($player instanceof Player){
+				if($player->isSurvival()) {
+					$count = $item->getCount();
+					if(--$count <= 0){
```
Comment (inxomnyaa) [inline src/pocketmine/block/ItemFrame.php:48]: And they still do this -.-"

**Pull request closed without merging**